pkg/monitors/collectd/couchbase/metadata.yaml

```diff
@@ -59,313 +161,1202 @@ monitors:
+      type: gauge
+      group: nodes
+    gauge.bucket.op.vb_active_queue_fill:
+      description: Number of active items per second being put on the active item
+        disk queue in this bucket
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.avg_disk_commit_time:
+      description: Average disk commit time.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.cpu_idle_ms:
+      description: CPU Idle milliseconds.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_views+2i_count:
+      description: Number of DCP views.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_2i_total_backlog_size:
+      description: Number of items in dcp backlog.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.vb_active_resident_items_ratio:
+      description: ratio of items kept in memory vs stored on disk
+      default: true
+      type: gauge
+      group: bucket
+    gauge.nodes.couch_views_actual_disk_size:
+      description: Amount of disk space occupied by Couch views (bytes).
+      default: false
+      type: gauge
+      group: nodes
+    gauge.bucket.op.couch_views_fragmentation:
+      description: How much fragmented data there is to be compacted compared to real
+        data for the view index files in this bucket.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_views+2i_total_backlog_size:
+      description: Number of views items remaining for replication.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.get_misses:
+      description: Number of get misses.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.vb_active_ops_create:
+      description: New items per second being inserted into active vBuckets in this
+        bucket.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_views+2i_items_sent:
+      description: Number of indexes items sent.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.vb_active_meta_data_memory:
+      description: Amount of active item metadata consuming RAM in this bucket.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_replica_producer_count:
+      description: Number of replica producers.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.bucket.op.ep_dcp_other_total_bytes:
+      description: Number bytes per second being sent for other DCP connections.
+      default: false
+      type: gauge
+      group: bucket
+    gauge.nodes.mcdMemoryAllocated:
+      description: Amount of memcached memory allocated (bytes).
+      default: false
+      type: gauge
+      group: nodes
+    gauge.storage.hdd.total:
+      description: Total harddrive space available to cluster (bytes)
```

Comment:
The way you did this one is right, this is not a sentence and so it shouldn't end in a period.